### DISCUSSION TITLE
1.4.0

### DISCUSSION THREAD

**Discussion by CamiloTerevinto** [CamiloTerevinto/TerevintoSoftware.StaticSiteGenerator#6] [Announcements]
A number of URL-related issues were fixed between versions 1.3.1 and 1.4.0. Release 1.4.0 adds a significant number of tests to ensure less breaking changes for future releases.

* Re-work how links are generated when localization is enabled - the default URL is still `/` but the rest of the links are generated with `{defaultCulture}/{link}`
* Fix several issues with kebab cased URLs. See [the multiple test cases added](https://github.com/CamiloTerevinto/TerevintoSoftware.StaticSiteGenerator/blob/main/tests/TerevintoSoftware.StaticSiteGenerator.Tests/Services/HtmlFormatterTests.cs).
* A small breaking change to the public API was done in 1.4.0 - `HtmlFormatter` and `UrlFormatter` were merged.

<hr /><em>This discussion was created from the release <a href='https://github.com/CamiloTerevinto/TerevintoSoftware.StaticSiteGenerator/releases/tag/v1.4.0'>1.4.0</a>.</em>

**Comment by deepakn1079** [comment]
it is not working with dotnet 7

**Reply by CamiloTerevinto** [reply]
Hey @deepakn1079, I've release version 2.0.0 that adds runtime support for .NET 7 and .NET 8 now
